bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/TransactionRpc.scala

```diff
@@ -148,7 +149,7 @@ trait TransactionRpc { self: Client =>
 
   def sendToAddress(
       address: BitcoinAddress,
-      amount: Bitcoins,
+      amount: CurrencyUnit,
```

Comment:
You have to be careful here, even though you are taking in a generic `CurrencyUnit` type, you need to convert it to `Bitcoins` _internally_ in this method as the bitcoind rpc server expects `Bitcoins`. For instance, if you passed in `Satoshis(1)` here and called `Satoshis(1).toBigDecimal` you would receive `1`, which would then bitcoind would interpret as one bitcoin!

```scala
@ val sat = Satoshis.one
sat: Satoshis = SatoshisImpl(Int64Impl(1))

@ sat.toBigDecimal
res5: BigDecimal = 1
```

You would effectively be sending one bitcoin when you meant to send one satoshi! Yikes!